snappybouncer/tasks.py

```diff
@@ -105,7 +105,7 @@ def update_snappy_ticket_with_extras(snappy_api, nonce, contact_key, subject):
     optout_url = settings.SITE_DOMAIN_URL + \
         "/controlinterface/subscription/?msisdn=" + \
         urllib.quote_plus(contact["msisdn"])
-    extra_info += "Opt this user out: " + optout_url + "\n"
```

Comment:
Please change to `Manage user subscriptions (opt out, unsubscribe or switch to baby):`